metagraph/src/cli/config/config.cpp

```diff
@@ -521,6 +523,25 @@ Config::Config(int argc, char *argv[]) {
     if (identity == COMPARE && fnames.size() != 2)
         print_usage_and_exit = true;
 
+    if (identity != BUILD && phase != 2) {
+        std::cerr << "Error: Phases are only supported for building. Remove --phase"
+                  << std::endl;
+        print_usage_and_exit = true;
+    }
```

Comment:
please add a check that phase is either 1 or 2, and print an error message otherwise